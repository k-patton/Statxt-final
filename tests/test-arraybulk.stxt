bool is_str_equal(string s1, string s2){
	int len_s1; 
	len_s1 =  strlen(s1);
	int len_s2;
	len_s2 = strlen(s2);
	
	if( len_s1 != len_s2) {
		return false;	
	}
	int i;
	for(i=0;i<len_s1;i++){
		if (strget(s1,i) != strget(s2,i)){
			return false;	
		}
	}
	return true; 
}

int main() {
	int[3] a;
	a = [3,2,4];
	print(a[1]);

	string[4] s;

	s = ["hello", "world", "yay", "what"];

	int i;
	for (i = 0; i < a[2]; i++) {
		printstr(s[i]);
	}

	if (is_str_equal(s[0], "hello")){
		printstr("yes");
	} else {
		printstr("no");
	}
}
