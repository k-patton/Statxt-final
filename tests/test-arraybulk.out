2
hello
world
yay
what
yes
